app/Services/Email.php

```diff
@@ -133,8 +134,6 @@ protected function parseLinks($string)
      */
     protected function setupEmail()
     {
-        $leaderboardVars = $this->getLeaderboardVars($this->users);
```

Comment:
For context, yes it gets stored in Flash, but only for the next `request` and then it's gone again. I didn't want to store for too long since it's meant to be data that might change quickly. Buuuut wanted to have it in flash for like if you loaded the leaderboard page, and then wanted to export the data of that page (no sense in doing all the heavy processing again for that same info). @deadlybutter not sure if each email cycle would be considered the same request or each is a unique one :shrug:... so flashed data may not be available. Just something to consider since I hadn't intended it to be used outside of a first and second request cycle.